pkg/api/controller_test.go

```diff
@@ -3826,27 +3826,33 @@ func TestController_MergeSquashing(t *testing.T) {
 
 	cases := []struct {
 		Name               string
-		Squash             bool
+		Squash             *bool
```

Comment:
I think adding a scenario where the parameter is not passed explicitly would have caught this.
Might want to add another flavor

Reply:
Great ide!  That's exactly what "default" does here :-)

Reply:
Missed that 😅